packages/cfd/src/Containers/__tests__/compare-accounts-content.spec.js

```diff
@@ -0,0 +1,179 @@
+        ).toBeInTheDocument();
+        expect(
+            screen.getByText(
+                /The spread is the difference between the buy price and sell price. A variable spread means that the spread is constantly changing, depending on market conditions. A fixed spread remains constant but is subject to alteration, at the Broker's absolute discretion./i
+            )
+        ).toBeInTheDocument();
+        expect(screen.getByText(/Deriv charges no commission across all account types/i)).toBeInTheDocument();
+        expect(
+            screen.getByText(
+                /You’ll get a warning, known as margin call, if your account balance drops down close to the stop out level/i
+            )
+        ).toBeInTheDocument();
+        expect(
+            screen.getByText(
+                /To understand stop out, first you need to learn about margin level, which is the ratio of your equity \(the total balance you would have if you close all your positions at that point\) to the margin you're using at the moment. If your margin level drops below our stop out level, your positions may be closed automatically to protect you from further losses./i
+            )
+        ).toBeInTheDocument();
+        expect(
+            screen.getByText(/Indicates the availability of cryptocurrency trading on a particular account./i)
+        ).toBeInTheDocument();
+    });
+
+    it('should render the correct content for the DerivX platform', () => {
+        render(
+            <CompareAccountsContent
+                {...mock_props}
+                landing_companies={all_landing_companies}
+                platform='dxtrade'
+                residence='uk'
+            />
+        );
+
+        expect(screen.getAllByText(/Account currency/i)[0]).toBeInTheDocument();
+        expect(screen.getAllByText('EUR/GBP')).toHaveLength(1);
+        expect(screen.getByText(/Number of assets/i)).toBeInTheDocument();
+        expect(screen.getByText('90+')).toBeInTheDocument();
+        expect(
+            screen.getByText('FX-majors (standard/micro lots), FX-minors, Commodities, Cryptocurrencies (except UK)')
+        ).toBeInTheDocument();
+    });
+
+    it('should not see the Financial STP column in the DerivX platform', () => {
+        render(<CompareAccountsContent landing_companies={all_landing_companies} {...mock_props} platform='dxtrade' />);
+        expect(screen.queryByText('Financial STP')).not.toBeInTheDocument();
+    });
+
+    it('should render Account currency even if the user is not logged in', () => {
+        render(
+            <CompareAccountsContent landing_companies={all_landing_companies} {...mock_props} is_logged_in={false} />
+        );
+        expect(screen.queryByText(/Account currency/i)).toBeInTheDocument();
+    });
+
+    it('should render the column head from CFDs to Financial if is_eu is enabled', () => {
+        render(<CompareAccountsContent {...mock_props} landing_companies={all_landing_companies} is_eu={false} />);
```

Comment:
The description says is_eu is enabled but we are passing fakse to is_eu. Is this expected?

Reply:
Its supposed to be true... Corrected